src/CMakeLists.txt

```diff
@@ -8,23 +8,15 @@ include(AddWindowsResources)
 configure_file(${PROJECT_SOURCE_DIR}/cmake/bitcoin-config.h.in config/bitcoin-config.h USE_SOURCE_PERMISSIONS @ONLY)
 include_directories(${CMAKE_CURRENT_BINARY_DIR} ${CMAKE_CURRENT_SOURCE_DIR})
 
-# TODO: After the transition from Autotools to CMake, the obj/ subdirectory
-#       could be dropped as its only purpose was to separate a generated header
-#       from source files.
```

Comment:
@theuni 

You might be interesting in reviewing this PR, as this comment was added at your request while working on the CMake staging branch, if I recall correctly :)